src/C2 구현/P01 JS1 왕실의 나이트.js

```diff
@@ -0,0 +1,22 @@
+function solution(position) {
+  const xArray = "abcdefgh";
+  const col = xArray.indexOf(position[0]);
+  const row = position[1] - 1;
+  const move = [
+    [-2, -1],
+    [-2, 1],
+    [-1, 2],
+    [1, 2],
+    [2, -1],
+    [2, 1],
+    [-1, -2],
+    [1, -2],
+  ];
+  const answer = move.filter((val) => {
+    const [y, x] = val;
+    return row - y >= 0 && row - y <= 7 && col - x >= 0 && col - x <= 7;
```

Comment:
```suggestion
  const answer = move.filter(([y, x]) => row - y >= 0 && row - y <= 7 && col - x >= 0 && col - x <= 7
```
바로 destructuring 하는것도 좋았을거 같아요 :+1:

Reply:
문제를 푸는 것 뿐 아니라 리팩토링도 신경 써야겠어요 😃